File: Kotlin/app/src/main/java/com/example/todolist/SplashActivity.java
package com.example.todolist;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

class SplashActivity extends AppCompatActivity{
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        getSupportActionBar().hide();

        final Intent i = new Intent(SplashActivity.this, MainActivity.class);
        new Handler().postDelayed(new Runnable(){
            @Override
            public void run(){
                startActivity(i);
                finish();
            }
        }, 1000);
    }
}

File: MobileDev_Kotlin/app/src/main/java/com/example/todolist/SplashActivity.kt
package com.example.todolist

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        getSupportActionBar().hide();

        final Intent i = new Intent(packageContext: SplashActivity.this.MainActivity.class);
        new Handler().postDelayed(new Runnable(){
            @Override
            public void run(){
                startActivity(i);
                finish();
            }
        }, delaymillis: 1000);
    }
}